examples/arp.rs

```diff
@@ -0,0 +1,16 @@
+extern crate pnetlink;
+
+use pnetlink::packet::netlink::NetlinkConnection;
+use pnetlink::packet::route::neighbour::{Neighbour,Neighbours};
+
+fn print_neighbour(neighbour: &Neighbour) {
+    println!("{:?}", neighbour);
```

Comment:
Hmm, ok, I can do that - can I ask why? As in - usability, or familiarity? For the former it makes a lot of sense to me to change. For the latter - this is showing more information than they tend to - e.g. arp and ip neighbor only show one family at a time - and I'd not really want to cripple it that way :)